src/dmd/backend/cgcs.d

```diff
@@ -144,19 +114,55 @@ void comsubs()
 /*******************************
  */
 
-void cgcs_term()
+public void cgcs_term()
 {
     vec_free(csvec);
     csvec = null;
     debug debugw && printf("freeing hcstab\n");
     //hcstab.dtor();  // cache allocation for next iteration
 }
 
+
+/***********************************************************************/
```

Comment:
I think `private:` acts as a natural separator and that the above is not needed.